api/client/gro_client.py

```diff
@@ -237,6 +238,67 @@ def convert_unit(self, point, target_unit_id):
         point['unit_id'] = target_unit_id
         return point
 
+    def get_region_info_given_lat_lon(self, latitude, longitude, output_region_level):
+    """Look up region_ids of the given level that contains each pair of latitude and longitude resembling spatial points.
+        Start from the continent level, find the continent_ids containing the given points' latitude and longitude
+        Then find corresponding country ids that contain these points, province ids and district ids.
+        The depth of spatial granuarity returned depends on the output_region_level argument.
+
+        Parameters
+        ----------
+        latitude : list of float
+        longitude: list of float
+        output_region_level : integer
+            The region level of interest. See REGION_LEVELS constant.
+
+        Returns
+        -------
+        Geopandas Spatial Point data frame with column names of region ids
```

Comment:
still returning the geopandas data frame because we need to keep the reference of which lat lon point refers to which region_id

Reply:
Since this is a fairly basic function, like get_descendant_regions, search and lookups etc., it's probably better to only use basic list rather than  dataframes in the input or output.  input a list of lat, lon pairs, and return a list of region ids.

Reply:
Done